packages/image_picker/lib/image_picker.dart

```diff
@@ -23,6 +23,21 @@ enum ImageSource {
   gallery,
 }
 
+/// Which camera to use when picking images/videos while source is `ImageSource.camera`.
+///
+/// Not every device supports both of the positions.
+enum CameraDevice {
```

Comment:
I would name this `CameraFacing`.

or if you anticipate there will be more properties going into here, you can make it a class with const ctr.

Reply:
Yeah, `CameraFacing` is probably a more straight forward name for the current situation.
I named this `CameraDevice` following the iOS naming. I think `CameraDevice` is more future proof as there might be different cameras other than front and rear in future. Maybe like a secondary rear camera that can be chosen as default camera.
WDYT?

Reply:
Sure. It was a nit anyway.